frb/builds/build_frbs.py

```diff
@@ -139,8 +139,8 @@ def frb_190102():
                         364.545 * units.pc / units.cm**3,
                         z_frb=0.2912) # Updated redshift
     # Error ellipse [REQUIRED]
-    frb190102.set_ee(0.16, 0.2, 0., 68.) # Statistical (Day+2020)
-    frb190102.set_ee(0.44, 0.5, 0., 68., stat=False) # Systematic
+    frb190102.set_ee(0.16, 0.2, 90., 68.) # Statistical (Day+2020)
```

Comment:
please use `theta` and `cl`.  In case we 
change the order of the keywords in code.

Here and throughout.